git_workflow.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Some projects use separate directory for workflows, such as here: https://github.com/SAP-samples/datahub-integration-examples/tree/main/GitWorkflow/scripts 
What do you think about it? 
It's probably unnecessary if we have only a one script, but it might be useful if we would add more in the future

Reply:
I also think it's good to separate the scripts.

Reply:
It was already merged in #119.
May be with next workflow